pkg/api/mongodb_repository.go

```diff
@@ -51,6 +51,10 @@ func (r *MongoRepository) FindByName(name string) (*Definition, error) {
 	defer session.Close()
 
 	err := coll.Find(bson.M{"name": name}).One(&result)
+	if err != nil {
+		return nil, ErrAPIDefinitionNotFound
```

Comment:
this can be mongo connection error - is there a way to check if this is real error or result error?